templates/crate/details.html

```diff
@@ -38,16 +38,6 @@
                                 {%- endif -%}
                             </li>
                         {%- endif -%}
-                        {# List the release author's names and a link to their docs.rs profile #}
```

Comment:
Could you show the owners instead?

Reply:
The owners are already shown at the bottom of this sidebar.

Reply:
Check https://github.com/rust-lang/docs.rs/blob/696ff69d1d8ab9b3f2794e88fa543284ad640d6b/templates/crate/details.html#L123